tests/test_oidc.py

```diff
@@ -99,6 +99,6 @@ def test_study_year(self):
             'sub': 'louis.vaneau.1829',
             'name': 'Louis Vaneau',
             'email': 'louis.vaneau.1829@polytechnique.org',
-            'study_year': 'X1829',
-            'grad_year': 1829,
+            'study_years': ['X1829'],
+            'grad_years': [1829],
```

Comment:
I feel having `grad_years` as only a list of numbers can lead to ambiguities (which value is what).

Why did we need it in addition of `study_years` again?

Reply:
Indeed this field is not used by current clients (we have ``promo_sortie`` in authgroupex that nobody really uses). Let's drop it!